src/custom/constants/ipfs.ts

```diff
@@ -0,0 +1,6 @@
+export const PINATA_API_KEY = process.env.REACT_APP_PINATA_API_KEY as string
+export const PINATA_SECRET_API_KEY = process.env.REACT_APP_PINATA_SECRET_API_KEY as string
```

Comment:
I've added both to github
![screenshot_2021-09-09_14-43-13](https://user-images.githubusercontent.com/43217/132766483-5e440c38-6127-4ef6-9ad0-c833c708386d.png)

Will trigger a redeploy and check whether it works :crossed_fingers: